### DISCUSSION TITLE
invalid_sso_token

### DISCUSSION THREAD

**Discussion by TuffyMouse** [timdorr/tesla-api#341] [Q&A]
Hi,
my code worked fine (based on the php example).
But since a couple of days a get the answer "invalid_sso_token".
I made no changes on my code.
Any idea?

Best Regards
TuffyMouse

**Comment by Liuxu1020** [comment]
have you resolved this problem?

**Reply by TuffyMouse** [reply]
I got it working for a few days. Now it's not working :-(

**Reply by Liuxu1020** [reply]
Ok，thanks
